Assignment-1_refactored/Main.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Email.h"
+#include "IEntity.h"
+#include <iostream>
+
+int main()
+{
+    int spamEmails, standardEmail, emailWithAttachments;
+    std::cout << "Enter the number of spam mails: ";
+    std::cin >> spamEmails;
+    std::cout << "Enter the number of standard mails: ";
+    std::cin >> standardEmail;
+    std::cout << "Enter the number of emails with attachments: ";
+    std::cin >> emailWithAttachments;
+    IEntity* entity = new Email(spamEmails, standardEmail, emailWithAttachments);
+    double footPrint = entity->calculateFootPrint();
```

Comment:
Variable name can be more descriptive, that can show its purpose like; "emailCarbonFootprint"